File: Guessing_Game/guess_game.go
package main

import (
	"fmt"
	"math/rand"
)

func main() {
	for{
		min, max := 111, 999
		guess := rand.Intn(max-min) + min
		var user int
		fmt.Print("Enter your guess between 1, 100: ")
		fmt.Scan(&user)
		if user == guess{
			fmt.Printf("You won the guess is %v\n",guess)
			break
		}else{
			fmt.Printf("You missed it, the guess is %v\n",guess)
		}
	}
}

File: Guessing_Game/guess_game.py
import random

def game():
    a = int(input("Enter lowest interval: "))
    b = int(input("Enter lowest interval: "))
    comp = random.randint(a, b)
    guess = int(input("Enter your guess: "))
    if guess == comp:
        print("YOu won. The value was ",comp)
        game()
    else:
        print("You loose. The value was ",comp)

game()